Add one link to `for Theme Designers.md` & re-order items on the page

chrisgrieser opened the pull request:
## Edited
Added only a link to my guide how to use the new 1.0 variables

## Checklist
- [x] before creating a new note, I searched the vault
- [x] new notes have the `.md` extension

claremacrae commented on 04 - Guides, Workflows, & Courses/for Theme Designers.md:
```diff
@@ -31,6 +31,7 @@ This note collects resources and guides for beginner and expert theme designers
 - [[Using Visual Studio Code for Theme Development]] by [[damiankorcz|Damian Korcz]]
 - [[Graph view customization|Styling the graph view]] 
 - [[Obsidian Design System Community File]] A community built, component and styles library in [[Figma]] for reference and use while designing user interfaces for Obsidian
+- [How To Work with Obsidian 1.0's new CSS Variables](https://www.youtube.com/watch?v=yl0pvIRTWWo)  by [[chrisgrieser|pseudometa]]
```
It's OK as it is, but how would you feel about moving it up to the 'Create a Custom Theme in Obsidian' link above and giving it the same movie icon - for consistency and to make it clear it's a video.

chrisgrieser replied:
did so in the last commit. going through the list, I noticed that the whole page could really use some better organization, so I re-ordered the list in a more reasonable manner. (no new content added)

claremacrae replied:
OK I can't review that in the website. The diffs are huge now.
Will look at it later on.

claremacrae replied:
Also, the summary in the PR no longer describes the change.

chrisgrieser replied:
oh right. edited the PR title to reflect the changes

claremacrae reviewed (commented):
The description says that items have been re-ordered - but when I try and compare the differences, the changes are wider than re-ordering, and I gave up actually doing a comparison.

There are some things I did notice though.

It has lost useful text like 'Ways to download all of Obsidian's theme source codes to learn from and search through'

At least one link has been removed:
[custom-css at Obsidian Forum](https://forum.obsidian.md/tag/custom-css)

Obsidian Design System Community File was intentionally put in to Guides the other day, whilst pairing with the author - it is not a tool (the tool is Figma).

chrisgrieser commented:
> The description says that items have been re-ordered - but when I try and compare the differences, the changes are wider than re-ordering, and I gave up actually doing a comparison.

yeah, I think diffs in general do not handle re-ordering very well.

> Oh, I forgot to mention, Ressources has a typo.

always forget it's spelled only with one s in English. fixed

> It has lost useful text like 'Ways to download all of Obsidian's theme source codes to learn from and search through'

there is already text at that link describing that it allows you to download all themes. I made the change to make the list more consistent, since with no other tool are the motivations for using the tool described. "ways" is also technically wrong, since there is only one such tool mentioned. 

> Obsidian Design System Community File was intentionally put in to Guides the other day, whilst pairing with the author - it is not a tool (the tool is Figma).

"it is not a tool" that's why I added "resources" to the heading 😉 
It's also not a guide really, since there aren't any instructions on how to use it, so I think it makes more sense in "Tools and Resources" to me.

> At least one link has been removed, as far as I can see:
[custom-css at Obsidian Forum](https://forum.obsidian.md/tag/custom-css)

oh yeah for got that one, I removed it since it's not really linking to anything that is guide, tool, or resource. it just links to a place where one could find snippets, which is really not much of help when you want to design a theme, aside from inspiration maybe. The only notable forum threads which give intros on how to do css yourself are already linked.

claremacrae commented:
> > The description says that items have been re-ordered - but when I try and compare the differences, the changes are wider than re-ordering, and I gave up actually doing a comparison.
> 
> yeah, I think diffs in general do not handle re-ordering very well.

Actually diffs do handle a pure re-ordering really well [*]. The difficulty comes when it's a single commit with re-ordering _and_ reformatting, _and_ removing lines: that's when the diffs become non-trivial to review.

So in this case I personally would have done the re-ordering as one commit, and the editing as another.

[*] By which I mean, they are trivially easy to review: 
- sort the lines in the original revision
- sort the lines in the new revision
- and diff the sorted lines
- any accidental deletion or duplication will be really obvious .

claremacrae commented:
Thanks @chrisgrieser!

@jsmorabito please note that when the vault is next published, the link to your Figma templates thing will be in the Tools and Resources section now...

chrisgrieser commented:
> So in this case I personally would have done the re-ordering as one commit, and the editing as another.
> 
> [*] By which I mean, they are trivially easy to review:
> 
> * sort the lines in the original revision
> * sort the lines in the new revision
> * and diff the sorted lines
> * any accidental deletion or duplication will be really obvious .

TIL. Will try to split up into multiple commits next time then! 👍

claremacrae commented:
> TIL. Will try to split up into multiple commits next time then! 👍

Thanks!

I think you know this, but for the avoidance of doubt, a single PR is still ideal - just split in to two commits (in this case...)
